lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
Hmm. I dunno, I think argument slicing will be pretty necessary anyway, given how few real constructors there are. Even those things with constructors have hidden parameters (e.g. ownerDocument on Text and friends).

Could you give an example where you think this will work out well?